### ISSUE TITLE
FileSystemOperationDocumentStorage fails at SaveInternalAsync when persisting query

### ISSUE THREAD

**Issue by sciocoder** [ChilliCream/graphql-platform#7631]
the issue was already reported #4139 and has not been resolved, it is also present in the current version 14.

```json
{
    "errors": [
        {
            "message": "Unexpected Execution Error",
            "extensions": {
                "message": "Cannot access a closed file.",
                "stackTrace": "   at System.IO.FileStream.FlushAsync(CancellationToken cancellationToken)\n   at HotChocolate.PersistedOperations.FileSystem.FileSystemOperationDocumentStorage.SaveInternalAsync(String filePath, IOperationDocument document, CancellationToken cancellationToken)\n   at HotChocolate.PersistedOperations.FileSystem.FileSystemOperationDocumentStorage.SaveInternalAsync(String filePath, IOperationDocument document, CancellationToken cancellationToken)\n   at HotChocolate.Execution.Pipeline.WritePersistedOperationMiddleware.InvokeAsync(IRequestContext context)\n   at HotChocolate.Execution.Pipeline.ReadPersistedOperationMiddleware.InvokeAsync(IRequestContext context)\n   at HotChocolate.Execution.Pipeline.DocumentCacheMiddleware.InvokeAsync(IRequestContext context)\n   at HotChocolate.Execution.Pipeline.TimeoutMiddleware.InvokeAsync(IRequestContext context)\n   at HotChocolate.Execution.Pipeline.ExceptionMiddleware.InvokeAsync(IRequestContext context)"
            }
        }
    ]
}
```

when IOperationDocument is of type OperationDocument the stream after [document.WriteToAsync](https://github.com/ChilliCream/graphql-platform/blob/6ddcc7c8697b83c32bba6ffd3afd3ec4a6f11a92/src/HotChocolate/PersistedOperations/src/PersistedOperations.FileSystem/FileSystemOperationDocumentStorage.cs#L96) is disposed [here](https://github.com/ChilliCream/graphql-platform/blob/6ddcc7c8697b83c32bba6ffd3afd3ec4a6f11a92/src/HotChocolate/Language/src/Language.SyntaxTree/Utilities/SyntaxPrinter.cs#L54)

**Comment by dcube9** [comment]
We have the same problem !

**Comment by michaelstaib** [comment]
This is now fixed and will be released with 14.1

**Comment by sciocoder** [comment]
> This is now fixed and will be released with 14.1

thanks!

**Comment by dcube9** [comment]
Great !
